server/models/migrations/v0.8.0-v0.9.0/setup-migration-script.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+
+# this script rebuild function and procedure 
+
+DATABASE="imck"
```

Comment:
By the way - a trick that will enable us to use this script in multiple sites and not just IMCK is to have the variable default to whatever is passed in.  It works like this:
```sh
DATABASE=${DATABASE:-"imck"}
```

What this will do is set DATABASE to whatever is passed in.  If nothing is passed in, it will default to "imck".  [Here is the documentation for these shell variables](https://bash.cyberciti.biz/guide/Default_shell_variables_value).

Reply:
That way, we can call this script:
```sh
DATABASE="pepkit" ./setup-migration-script.sh
```
and it will switch to operating on the database called "pepkit".